docs/whats_new/v1.3.0.md

```diff
@@ -183,7 +185,21 @@ Internally, we completed a major refactor of the core classes and introduced sta
 
 ## Multimodal
 
+### New Features
+AutoGluon's MultiModal module has been enhanced with a comprehensive "Bag of Tricks" update that significantly improves performance when working with combined image, text, and tabular data through advanced fusion techniques, data augmentation, and an integrated ensemble learner now accessible via a simple use_ensemble=True parameter.
```

Comment:
This doesn't work out of the box. It needs special manual steps to be taken to get the right dependencies so that it works.

Reply:
Need to document this

Reply:
Added a pointer to instruction of downloading checkpoints